lib/std/debug.zig

```diff
@@ -656,7 +656,7 @@ pub const StackIterator = struct {
         // The implementation of DWARF unwinding on aarch64-macos is not complete. However, Apple mandates that
         // the frame pointer register is always used, so on this platform we can safely use the FP-based unwinder.
         if (builtin.target.isDarwin() and native_arch == .aarch64)
-            return init(first_address, context.mcontext.ss.fp);
+            return init(first_address, @intCast(context.mcontext.ss.fp));
```

Comment:
```suggestion
            return init(first_address, @truncate(context.mcontext.ss.fp));
```

Actually, let's make this an explicit truncation. While we may be targeting an ILP32 ABI, it's still possible (though very unlikely) that random nonsense could be in the upper 32 bits of the FP register. We should just discard those bits rather than trigger UB.